docs/core/START_NEW_APPLICATION.md

```diff
@@ -13,15 +13,24 @@ Start a new Otter application.
   * This is used to run tests and generate distributable files. We strongly encourage to use an up-to-date LTS version of Node.js to ensure the support of all the Otter packages.
     Each package comes with a minimum Node.js version range defined in the `engine` property of its package.json file.
 
-## Creating a new Otter application
+## Creating a new Otter-based project
 
 ```shell
-# Create new application
 npm create @o3r my-project
-# Or
-yarn create @o3r my-project
+# Or a project using yarn package manager
+npm create @o3r my-project -- --package-manager=yarn
 ```
 
+### Generate a new application in the project
+
+After creating the project, you can generate a new application within it using the following command:
+
+```shell
+ng generate application my-webapp
```

Comment:
This should probably be `yarn ng` or `npm run ng`, otherwise there is a chance that the output will be `ng is not recognized...` or it will use the global `@angular/cli`

Reply:
yes, I know, I wanted something generic. I think we can stick with `npm` everywhere in our doc. Yarn users will know how to adapt the command :-)